docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -38,6 +40,14 @@ Each file type has a custom section (for example, `[Container]`) that is handled
 other sections will be passed on untouched, allowing the use of any normal systemd configuration options
 like dependencies or cgroup limits.
 
+For rootless containers, when administrators place quadlet files in the
+/etc/containers/systemd/users directory, all users sessions will execute the
+quadlet when the login session begins. If the administrator places a quadlet
+file in the /etc/containers/systemd/user/USERNAME/ directory, then only the
```

Comment:
@rhatdan can you elaborate on USERNAME over UID?  I "feel" that UID may be better suited but cannot really put my finger on it.

Reply:
I am fine with either, although Username is more human friendly.

Reply:
Switched to UIDs.